Gemfile

```diff
@@ -4,5 +4,8 @@ group :test do
   gem 'rake'
   gem 'rspec'
   gem 'beaker', '~>1.17.0'
+  if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
```

Comment:
That's true, but I think Bundler would choke before it reaches this point if the protocol portion of the URI was missing or if there were extra characters at the end.